File: app/src/main/java/com/example/utente10/galileo/bean/Beacon.java
package com.example.utente10.galileo.bean;

import com.google.android.gms.maps.model.LatLng;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;

@RealmClass
class Beacon implements RealmModel{
    @PrimaryKey
    String label;
    LatLng coordinates;
}

File: app/src/main/java/com/example/utente10/galileo/bean/Beacon.kt
package com.example.utente10.galileo.bean

import com.google.android.gms.maps.model.LatLng
import io.realm.RealmModel
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass

@RealmClass
data class Beacon (@PrimaryKey val label: String,
                   val coordinates: LatLng) : RealmModel